### ISSUE TITLE
gradle dependency not working

### ISSUE THREAD

**Issue by kaustav07** [jjobes/SlideDayTimePicker#2]
i added  `compile project(':slideDayTimePicker')` in my build.gradle in app module..but its failing sync

**Comment by RegisAG** [comment]
Same issue here. Any solution?

**Comment by kaustav07** [comment]
no, actually i think the owner of this repo haven't actually distributed this library through maven or jitpack or any other distribution system, i actually found some other library to work with, but if you want this one then just download all files and create a new module in your project and copy all the files there and then this dependency will work as the project would be in your work-space.
